common/lanelet2_extension/include/lanelet2_extension/regulatory_elements/CarmaTrafficSignal.h

```diff
@@ -53,6 +53,12 @@ enum class CarmaTrafficSignalState {
   CAUTION_CONFLICTING_TRAFFIC=9
 };
 
+namespace time{
+
+const int INFINITY_END_TIME_FOR_NOT_ENOUGH_STATES = 2147483647;
```

Comment:
This can be declared `constexpr` and should use `std::numeric_limits` (I assume this is indended to be the maximum int value):
```cpp
constexpr auto INFINITY_END_TIME_FOR_NOT_ENOUGH_STATES{std::numeric_limits<int>::max()};

Reply:
I think the date is for unix time maximum representable day https://en.wikipedia.org/wiki/Unix_time

Reply:
I'm not sure what you mean by this. Can you elaborate?